app/components/extra_mobile_data_request_status_details_component.html.erb

```diff
@@ -0,0 +1,70 @@
+      <%- else %>
+        <%= govuk_link_to('Make new request', responsible_body_internet_mobile_extra_data_requests_type_path) %>
+      <%- end %>
+    </div>
+  <%- elsif @extra_mobile_data_request.problem_not_eligible_status? %>
+    <div class="govuk-inset-text">
+      <h2 class="govuk-heading-m"><%= @extra_mobile_data_request.mobile_network.brand %> told us this account is not eligible</h2>
+
+      <p>This could be because:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>they are a new <%= @extra_mobile_data_request.mobile_network.brand %> customer</li>
+        <li>they do not meet <%= @extra_mobile_data_request.mobile_network.brand %>'s criteria</li>
+        <li>they already have fixed line broadband at home</li>
+      </ul>
+
+      <p>You can still:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>confirm this person needs internet access</li>
+        <li>check if there’s another mobile account they could use</li>
+        <li>request a <%= govuk_link_to('4G wireless router instead', how_to_request_4g_wireless_routers_path) %></li>
+      </ul>
+    </div>
+  <%- elsif @extra_mobile_data_request.problem_no_match_for_account_name_status? %>
+    <div class="govuk-inset-text">
+      <h2 class="govuk-heading-m"><%= @extra_mobile_data_request.mobile_network.brand %> did not recognise this name</h2>
+
+      <p>Check the following:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>the correct account holder was given</li>
+        <li>the name matches the name on the bill</li>
+      </ul>
+
+      <%- if @context == :school %>
+        <%= govuk_link_to('Make new request', extra_data_requests_type_internet_mobile_school_path(@extra_mobile_data_request.school)) %>
+      <%- else %>
+        <%= govuk_link_to('Make new request', responsible_body_internet_mobile_extra_data_requests_type_path) %>
+      <%- end %>
+    </div>
+  <%- elsif @extra_mobile_data_request.problem_other_status? %>
+    <div class="govuk-inset-text">
+      <h2 class="govuk-heading-m"><%= @extra_mobile_data_request.mobile_network.brand %> couldn’t process this request</h2>
+
+      <p>They did not give a reason why.</p>
+      <p>You can still:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>confirm this person needs internet access</li>
+        <li>check if there’s another mobile account they could use</li>
+        <li>request a <%= govuk_link_to('4G wireless router instead', how_to_request_4g_wireless_routers_path) %></li>
+      </ul>
+    </div>
+  <%- elsif @extra_mobile_data_request.unavailable_status? %>
+    <div class="govuk-inset-text">
+      <h2 class="govuk-heading-m"><%= @extra_mobile_data_request.mobile_network.brand %> is not offering data increases yet</h2>
+      <p>We cannot request an increase in data from a network that’s not participating in the offer.</p>
```

Comment:
I'm surprised these paragraphs don't need `govuk-body` classes on them 🤔

Reply:
I copied them from the prototype pretty much as-is - but sure, I'll add those classes (done in bb5535cc)